.github/workflows/docs.yml

```diff
@@ -0,0 +1,27 @@
+name: Build and publish Auto generated API documentation
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
```

Comment:
Double check if this is needed. IIRC It's in the build script for running on macos

Reply:
It might not be necessary, but I would rather be explicit about the version of node that it is using.